### DISCUSSION TITLE
T in Time Series Evaluation Metrics

### DISCUSSION THREAD

**Discussion by alirezatwk** [autogluon/autogluon#4229] [Q&A] [answered]
On [this page](https://auto.gluon.ai/stable/tutorials/timeseries/forecasting-metrics.html), for the definition of T it says "length of the observed time series". Based on the following image on [this page](https://auto.gluon.ai/stable/tutorials/timeseries/forecasting-indepth.html), I guess the observed time series is the partition of the time series that is used for training our model. 
![image](https://github.com/autogluon/autogluon/assets/44779572/a9ddb784-831b-48c5-a804-beac1d731952)

My question is what if the length of our time series are different from each other? I mean shouldn't it be T_i instead of T? T_i is the length of observed ith time series.

**Comment by shchur** [comment, marked as answer]
Yes, you are absolutely right, a more precise definition of the metrics would use $T_i$ in case time series have different lengths. We used $T$ in all formulas to avoid clutter and make the formulas more readable.

**Reply by alirezatwk** [reply]
Thanks for your answer.
